website/docs/migration/1.0.0_to_2.0.0.mdx

```diff
@@ -0,0 +1,246 @@
+
+But with `Notifier` (and its `FamilyNotifier` variant), no pipe-down is ever needed.  
+The `.family` modification, which is very important, is clearly stated in its own definition,
+and it's used directly in its `build` method.  
+The whole thing is more readable, more concise and less error-prone.
+<AutoSnippet language="dart" {...familyAndDispose}></AutoSnippet>
+
+Notice how the code-generated version is even more concise and less error-prone than ever.
+
+:::info
+`Notifier` and `AsyncNotifier` are intrinsically bound to the `.autoDispose` and `.family` features.
+Enabling or disabling such features means it's needed to change the subclassed notifier type.
+
+For example, to define an `AsyncNotifier` with `.autoDispose` and `.family` features altogether,
+it is needed to extend `AutoDisposeFamilyAsyncNotifier`.
+:::
+
+### Lifecycles (and `dispose`) have a different behavior
+Lifecycles  between `Notifier` (and `AsyncNotifier`) and `StateNotifier` (and `ChangeNotifier`)
+differ substantially.
+
+A `StateNotifier` (or a `ChangeNotifier`) instance is indeed bound to their outer
+`StateNotifierProvider` (or `ChangeNotifierProvider`).  
+When the latter rebuilds, their exposed value, aka its `StateNotifier` (or `ChangeNotifier`)
+is `disposed`, and then re-created (i.e. its constructor gets invoked again).
+
+Take this example:
+```dart
+class MyNotifier extends StateNotifier<int> {
+  MyNotifier(this.period) : super(0) {
+    _timer = Timer.periodic(period, (t) => update());
+  }
+  final Duration period;
+  late final Timer _timer;
+
+  void update() => state++;
+
+  @override
+  void dispose() {
+    _timer.cancel();
+    super.dispose();
+  }
+}
+
+final myNotifierProvider = StateNotifierProvider<MyNotifier, int>((ref) {
+  final Duration period = ref.watch(durationProvider);
+  return MyNotifier(period);
+});
+```
+
+A first takeaway from this example is the following: *if `durationProvider` updates,
+`MyNotifier` disposes*.  
+As stated above, its instance is then re-created and its internal state re-initialized as usual.
+
+The second takeaway is that, unlike every other provider, the `dispose` callback is to be defined
+in the class, separately.  
+It is still possible to write `ref.onDispose` in its *provider*,
+but two different dispose callbacks are to be maintained.
+
+This API ambiguity is solved with `Riverpod 2.0`.  
+Check the above example, migrated to its `Notifier` equivalent.
+<AutoSnippet language="dart" {...oldLifecycles}></AutoSnippet>
+
+Here, instead, `MyNotifier`, will *not* dispose when `durationProvider` updates:
+only its *internal state* will.  
+This breaking change simplifies the API and, hopefully, your DX, for two reasons:
+1. there is only one lifecycle to take care of, the one we really care about: `Notifier`'s inner state;
+2. there is only one API to register a `dispose` action: `ref.onDispose`.
+
+### There is no `.mounted` property
+WIP
+
+### Final remarks
```

Comment:
This heading feels superfluous. I'd remove it and directly talk about the other things